plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -352,20 +356,29 @@ private TestExecutionResult testPluginAgainst(MavenCoordinates coreCoordinates,
 			System.err.println("Error : " + e.getMessage());
 			throw new PluginSourcesUnavailableException("Problem while checking out plugin sources!", e);
 		}
-		
+        
         List<String> args = new ArrayList<String>();
         boolean mustTransformPom = false;
         // TODO future versions of DEFAULT_PARENT_GROUP/ARTIFACT may be able to use this as well
-        if (pomData.parent.groupId.equals("com.cloudbees.jenkins.plugins") && pomData.parent.artifactId.equals("jenkins-plugins") ||
+        final MavenCoordinates parent = pomData.parent;
+        final boolean isCB = parent.matches("com.cloudbees.jenkins.plugins", "jenkins-plugins") ||
                 // TODO ought to analyze the chain of parent POMs, which would lead to com.cloudbees.jenkins.plugins:jenkins-plugins in this case:
-                pomData.parent.groupId.equals("com.cloudbees.operations-center.common") && pomData.parent.artifactId.equals("operations-center-parent") ||
-                pomData.parent.groupId.equals("com.cloudbees.operations-center.client") && pomData.parent.artifactId.equals("operations-center-parent-client")) {
+                parent.matches("com.cloudbees.operations-center.common", "operations-center-parent") ||
+                parent.matches("com.cloudbees.operations-center.client", "operations-center-parent-client");
+        final boolean pluginPOM = parent.matches("org.jenkins-ci.plugins", "plugin");
+        final boolean coreRequiresNewParentPOM = coreCoordinates.compareVersionTo(CORE_NEW_PARENT_POM) >= 0;
+        if ( isCB || (pluginPOM && parent.compareVersionTo("2.0") >= 0)) {
             args.add("-Djenkins.version=" + coreCoordinates.version);
-            args.add("-Dhpi-plugin.version=1.99"); // TODO would ideally pick up exact version from org.jenkins-ci.main:pom
+            args.add("-Dhpi-plugin.version=1.117"); // TODO would ideally pick up exact version from org.jenkins-ci.main:pom
+            // There are rules that avoid dependencies on a higher java level. Depending on the baselines and target cores
+            // the plugin may be Java 6 and the dependencies bring Java 7
+            args.add("-Denforcer.skip=true");
+        } else if (coreRequiresNewParentPOM && pluginPOM && parent.compareVersionTo("2.0") < 0) {
+            throw new RuntimeException("New parent POM required for core >= 1.646");
```

Comment:
Since you have already defined it you could use the `CORE_NEW_PARENT_POM` constant here. Not important though.